api/swagger.yaml

```diff
@@ -1267,7 +1267,31 @@ definitions:
       Containers:
         x-nullable: false
         type: "integer"
-
+      Platforms:
+        type: "object"
+        description: "An array containing all platforms supported by the image"
+        items:
+          type: "object"
+          properties:
+            Architecture:
+              type: "string"
+            OS:
+              description: "operating system"
+              type: "string"
+            OSVersion:
```

Comment:
`OSVersion` was deprecated, as well.

Reply:
Wait, doesn't Windows use and rely on `os.version`? (I think the "relying on it" part was still a TODO, but was definitely planned, especially with the imminent RS3 that breaks compat)

Reply:
Yeah, +1

Reply:
We asked for commentary and specification on this field from the MSFT side and ended up deprecating the field because we had no response. Perhaps, an annotation would be sufficient here?

Reply:
See https://github.com/docker/distribution/issues/2398#issuecomment-333659799 for where it was confirmed that `os.version` is not in fact deprecated:

> Actually, it was left in place, apologies for the confusion.
> 
> The issue I see is that it is under-specified. What lands in that field? Do versions have an ordering? What is implied by that ordering? It would be good to have the microsoft folks work with OCI to define what is actually needed here, rather than have us guess.